src/EPR.Accreditation.Facade.Common/RESTservices/HttpAccreditationService.cs

```diff
@@ -90,6 +90,13 @@ public async Task<Guid> CreateAccreditation(Dtos.Accreditation accreditation)
             return await Get<List<AccreditationTaskProgress>>($"{accreditationExternalId}/TaskProgress");
         }
 
+        public async Task<AccreditationMaterial> GetLastCalendarYearWaste(
```

Comment:
There's already functionality that does waste last year: AccreditationMaterialService.GetReprocessedWasteLastYear